internal/proxy/proxy.go

```diff
@@ -196,13 +202,22 @@ func (c *portConfig) nextDBPort(version string) int {
 	}
 }
 
-// Client represents the state of the current instantiation of the proxy.
+// Client proxies connections from a local client to the remote server side
+// proxy for multiple Cloud SQL instances.
 type Client struct {
 	cmd    *cobra.Command
 	dialer cloudsql.Dialer
 
 	// mnts is a list of all mounted sockets for this client
 	mnts []*socketMount
+
+	// connCount tracks the number of all open connections from the Client to
+	// all Cloud SQL instances.
+	connCount uint64
```

Comment:
Do we need to ensure this is 64bit aligned to use atomic.sync? 

If so, should we add a test to ensure it doesn't change? https://github.com/GoogleCloudPlatform/cloudsql-proxy/pull/550/files#diff-de11c0b4b635f0cda6d02b369cf94915ec3595d09d5115753f8e9d9d19dcf0dcR290-R296